Safeway.h

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include "LeakedObjectDetector.h"
+#pragma once
```

Comment:
`#pragma once` should be the first line written in this header file
- [ ] revise